src/onboarding/success/OnboardingSuccessScreen.tsx

```diff
@@ -4,24 +4,15 @@ import { Image, StyleSheet, Text, View } from 'react-native'
 import { background } from 'src/images/Images'
 import Logo from 'src/images/Logo'
 import { nuxNavigationOptionsNoBackButton } from 'src/navigator/Headers'
+import { navigate } from 'src/navigator/NavigationService'
 import { Screens } from 'src/navigator/Screens'
-import { goToNextOnboardingScreen, onboardingPropsSelector } from 'src/onboarding/steps'
-import { useSelector } from 'src/redux/hooks'
 import colors from 'src/styles/colors'
 import { typeScale } from 'src/styles/fonts'
 import { Spacing } from 'src/styles/styles'
 
 function OnboardingSuccessScreen() {
-  const onboardingProps = useSelector(onboardingPropsSelector)
   useEffect(() => {
-    const timeout = setTimeout(
-      () =>
-        goToNextOnboardingScreen({
-          firstScreenInCurrentStep: Screens.VerificationStartScreen,
-          onboardingProps,
-        }),
-      3000
-    )
+    const timeout = setTimeout(() => navigate(Screens.ChooseYourAdventure), 3000)
```

Comment:
Wonder if we should make CYA optional through the onboarding config. Thoughts?

Reply:
I don't think so. with your refactor it will be very easy to take out anyways when/if product wants us to

Reply:
Yeah I think it works for the forking model for runtime, but not sure about the modular runtime. Thoughts @jeanregisser? I guess it can be done separately if required.

Reply:
ah yes it seems we may need this to be configurable.
Is CYA present in the Beefy app?

Reply:
No CYA for beefy